components/Vote/ElectionCandidacy.js

```diff
@@ -110,6 +110,7 @@ const fields = (t, vt) => ([
   },
   {
     label: t('profile/disclosures/label'),
+    explanation: t('profile/disclosures/explanation'),
```

Comment:
Probably should be wrapped in `<Label>`: `<Label>{t('profile/disclosures/explanation')}</Label>` (The reason not all explanation get auto wrapped is flexibility—any react node can be passed—strings are valid too but probably we want the small grey type here)

Reply:
It should and it will.